shared/helpers/package.json

```diff
@@ -17,6 +17,8 @@
     "@axe-core/puppeteer": "^4.8.5",
     "@govuk-frontend/config": "*",
     "@govuk-frontend/lib": "*",
+    "@testing-library/jest-dom": "^6.4.5",
+    "@types/testing-library__jest-dom": "^6.0.0",
```

Comment:
**nitpick** Given the `jest.config.mjs` is at the root of the repository, I think those two should go in the root `package.json`. That's also where `jest` and its plugins are listed as dependencies. I think things work at the moment because Node [hoists packages to the `root` node_modules when it installs dependencies from workspaces](https://leticia-mirelly.medium.com/a-comprehensive-guide-to-npm-workspaces-and-monorepos-ce0cdfe1c625#979b).